lib/cc/engine/analyzers/analyzer_base.rb

```diff
@@ -9,7 +9,11 @@ def initialize(engine_config:, directory:)
 
         def run
           files.map do |file|
-            process_file(file)
+            begin
+              process_file(file)
+            rescue => e
+              $stderr.puts e
```

Comment:
I'm fine with the simpler `puts e` syntax and omitting the backtrace. I think you should keep the "skipping file" line.

WDYT @mrb ?